source/data/tier1data/features/collaboration.rst

```diff
@@ -0,0 +1,192 @@
+.. _collaboration: 
+
+##############################
+Sharing data and collaboration
+##############################
+
+Depending on your research project, different people might need different access to the data stored in Tier-1 Data.
+
+Some projects are very open: everyone can work on all data in the project.  
+Other projects, especially those with sensitive data, might need to be more strict in who can access which data and in which manner.
+Tier-1 Data allows you to manage this in detail by assigning different levels of access to *groups*, which can consist of any number of users.
+
+****************
+Groups and users
+****************
+In Tier-1 Data, individual users can be put together in groups. 
+Currently, we distinguish between two groups: project groups and access groups.  
+
+A **project group** is created when a research group applies for a project in Tier-1 Data.
+It includes all collaborators on the project, some of which are **responsibles**, and has a shared collection.
+
+**Access groups** are subgroups of a project group and are used to assign specific permissions to certain users based on their roles.
+For example, data providers and data analysts may need different permissions on the datasets of the project. By creating an access group for data providers
+with the users that have this role you can define which permissions all data providers have based on that role (e.g. writing data to a certain collection).
+The same goes for data analysts, who would need read-only access to the datasets but write-access to a collection for results.
+A user that has both roles will have both sets of permissions. This way, permissions are not assigned to individuals based on who they are but based on their role in the project.
+
+For example:
+
+- Chemistry (project group)  
+  - Mary
+  - John
+  - Chris
+- Chemistry_data_providers (access group)  
+  - John
+- Chemistry_data_analysts (access group)  
+  - Mary
+  - Chris 
+
+Permissions in Tier-1 Data can be given to either individual users or groups.
```

Comment:
I think this is confusing. Maybe mention that depending on the client persmissions can be applied to users and groups but that in general the recommended way to work is to set permissions based on roles and groups

Reply:
Applied